scss/contact/contact.scss

```diff
@@ -14,50 +14,62 @@
     padding: 2%;
     width: 80%;
     background-color: #dfd3c3;
-    border-radius: 5px;
+    border-radius: 10px;
     margin-bottom: 5%;
     margin-top: 5%;
     font-family: 'Cormorant', serif;
-    color: #000;
+    overflow: hidden;
+    font-size: 20px;
+    font-weight: 400;
+    position: relative;
 
     h1 {
         text-align: center;
         color: black;
         font-size: 30px;
+        font-weight: 700;
     }
 
     input {
-        border: #000;
         padding: 10px;
         margin: 1%;
         width: 95%;
-        border-radius: 5px;
+        display: block;
+        // border-radius: 10px;
     }
 
     label {
-        margin: 1%;
-        border-radius: 5px;
+        display: inline-block;
+        margin-left: 10px;
+        position: relative;
     }
 
     textarea {
-        border: #000;
         padding: 10px;
         margin: 1%;
         width: 95%;
         height: 200px;
-        border-radius: 5px;
+        // border-radius: 10px;
```

Comment:
If you don't need this code then you might want to delete it

Reply:
Tabbing is not the only thing you need to clean up before submitting a pull request, unused code like this also needs to be cleaned up.